### DISCUSSION TITLE
VS1015 Install

### DISCUSSION THREAD

**Discussion by mbhansen6** [codecadwallader/codemaid#1018] [Q&A]
I am trying to install the extension for Visual Studio 2015 (CodeMaid v10.6), but because I have Visual Studio 2022 installed I get an error "This extension is not installable on any currently installed products" and the log references Visual Studio 2022.  Is there a way to install CodeMaid for all instances of Visual Studio - 2015, 2019, 2022?

**Comment by codecadwallader** [comment]
Visual Studio 2022 had some breaking changes for extensions, so it had to be separated out to a new installer with a new id.  I believe it can be installed alongside all of the previous versions (VS2019 and backwards) but since there are different CodeMaid installers with the same extension identifier between VS2015 and VS2019 I'm not sure if those can be installed side-by-side.
 
VS2015: https://github.com/codecadwallader/codemaid/releases/download/v10.6/CodeMaid.v10.6.144.vsix
VS2019: https://marketplace.visualstudio.com/items?itemName=SteveCadwallader.CodeMaid
VS2022: https://marketplace.visualstudio.com/items?itemName=SteveCadwallader.CodeMaidVS2022
